src/table/column-widths-utils.ts

```diff
@@ -40,3 +40,6 @@ function checkProperty(column: TableProps.ColumnDefinition<any>, name: 'width' |
     );
   }
 }
+
+export const treatAsNumber = (value: number | string | undefined) =>
```

Comment:
Not sure if it is an issue in this context but `treatAsNumber(NaN)` and `treatAsNumber('NaN')` produce true.

Reply:
good catch, fixed for completeness